src/analytij/data.clj

```diff
@@ -48,6 +48,7 @@
   - start date
   - end date
   - metrics
+  - view-id
```

Comment:
@uswitch If you decide not to merge the PR, please take at least this line as it updates your documentation to the reality of the validation constraints(;